orchestra/tests/helpers/fixtures.py

```diff
@@ -235,7 +240,8 @@ def setup_models(test_case):
     for task_slug, details in tasks.items():
         task = TaskFactory(project=test_case.projects[details['project_name']],
                            step_slug=test_case.test_step_slug,
-                           status=details['status'])
+                           status=details['status'],
+                           start_datetime=parse('2015-10-12T01:00:00+00:00'))
```

Comment:
hm, I'd still rather see `timezone.makeaware(datetime(year=2015, month=10, day=12, hour=1), timezone.utc)`, but I'm getting really tired of trying to care about timestamps, so also feel free to leave it.